### DISCUSSION TITLE
preview server not finding the hostname

### DISCUSSION THREAD

**Discussion by xhangy-5** [observablehq/framework#1535] [Q&A] [answered]
Hi all, 

after i run npm init @observablehq to build up the dependency, and i run "npm run dev" why the local host preview page can not properly launch?
![image](https://github.com/user-attachments/assets/3bff6809-67d2-499c-afd3-f16d844106d3)

**Comment by Fil** [comment, marked as answer]
Can you try and pass the host name explicitly, with the following command:

> npm run dev -- --host=127.0.0.1

**Reply by Fil** [reply]
Also can you describe your environment (OS and shell)?

**Reply by xhangy-5** [reply]
thanks, it works~!!! also i closed my VPN

**Comment by Fil** [comment]
Issue tracked as https://github.com/observablehq/framework/issues/1536.
